driver/generic/indev_example.py

```diff
@@ -0,0 +1,34 @@
+# This is the minimal pure-python lvgl pointer-type driver. It does not read 
+# any hardware and instead just presses at location (50,100)
+#
+# In a real driver, always return the last position in data.point, even if 
+# the data.state goes to 'released' (lv.INDEV_STATE.REL), so maybe keep that
+# in self.point.
+#
+# The 'return False' just means there is no further data to read, which is
+# what you want for a pointer-type driver.
+#
+# To use your driver (after you've made it read some real hardware):
+# 
+#       from my_indev import my_indev
+#       touch = my_indev()
+
+
+import lvgl as lv
+
+class my_indev:
+
+    def __init__(self):
+        indev_drv = lv.indev_drv_t()
```

Comment:
I think it's important to keep a reference to `indev_drv_t` and prevent gc from collecting it.  
The simplest way it to keep it as a member of `self`, as done here:

https://github.com/lvgl/lv_binding_micropython/blob/dadad31bdce92fc96ffac6b2805ac0d5d4b884eb/driver/linux/evdev.py#L46

Reply:
done